application/build.gradle

```diff
@@ -39,6 +39,7 @@ shadowJar {
 
 dependencies {
     implementation project(':database')
+    implementation project(':tag-system')
```

Comment:
why is it its own "project", we did not do this to the other commands as well, did we? whats the benefit? maybe others should do that as well?

Reply:
Yes, I was planning on doing a PR for the command system after this one. It's so that you don't have a big blob of packages in a single module; the "application" module should only contain the `Application` enum.